README.md

```diff
@@ -18,11 +18,13 @@ DISCLAIMER - Postgres must be installed in your system for the following steps.
     }
 
     module.exports.initial = initial;
-6: npm run dataGenPG
+6: npm run dataGen
   Please wait til you see the following: finished data generation: X seconds
 7: npm run createPG
   Please wait til you see the following messages: Database successfully created!
   Table successfully created!
 8: npm run seedPG
   Please wait til you see the following: finished seeding Postgres database: X seconds
```

Comment:
Nice signals to the developer to let them know whether things are going right or not!